script/import_scripts/base/generic_database.rb

```diff
@@ -350,6 +488,29 @@ def create_post_table
           post_id #{key_data_type} NOT NULL PRIMARY KEY
         )
       SQL
+    end
+
+    def create_pm_post_table
+      @db.execute <<-SQL
+        CREATE TABLE IF NOT EXISTS pm_post (
+          id #{key_data_type} NOT NULL PRIMARY KEY,
+          raw TEXT,
+          topic_id #{key_data_type} NOT NULL,
+          user_id #{key_data_type} NOT NULL,
+          created_at DATETIME,
+          reply_to_post_id #{key_data_type},
+          url TEXT,
+          upload_count INTEGER DEFAULT 0
+        )
+      SQL
+
+      @db.execute 'CREATE INDEX IF NOT EXISTS post_by_user_id ON post (user_id)'
+
+      @db.execute <<-SQL
+        CREATE TABLE IF NOT EXISTS post_order (
+          post_id #{key_data_type} NOT NULL PRIMARY KEY
+        )
```

Comment:
I think the index and table aren't needed, otherwise it should reference the `pm_post` table.